app/utils/broadcast.py

```diff
@@ -0,0 +1,267 @@
+import pyproj
+from flask import render_template, url_for
+from postcode_validator.uk.uk_postcode_validator import UKPostcode
+from shapely import Point
+from shapely.geometry import Polygon
+from shapely.ops import unary_union
+
+from app.broadcast_areas.models import CustomBroadcastArea
+from app.formatters import round_to_significant_figures
+from app.main.forms import (
+    CartesianCoordinatesForm,
+    DecimalCoordinatesForm,
+    PostcodeForm,
+)
+
+
+def create_coordinate_area_id(coordinate_type, first_coordinate, second_coordinate, radius):
```

Comment:
Is this really an ID, isn't it more of a name/slug?

If this is being used as an ID then most of the content is likely to be unchanging which is a pretty large waste of entropy space (And in fact wastage like this caused issues for an MNO when we submitted 10 Polygons as it made one of the strings the software they used as an ID overflow the string).